package.json

```diff
@@ -72,6 +72,9 @@
     "node": ">=16.0.0"
   },
   "productName": "DeltaChat",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
what's this needed for?

Reply:
same as before, I forgot to remove it haha